src/ansys/additive/core/additive.py

```diff
@@ -160,6 +161,9 @@ def __init__(
             server_connections, host, port, nservers, product_version, self._log, linux_install_path
         )
         self._nsims_per_server = nsims_per_server
+        if self._nsims_per_server > 1:
+            self.apply_server_settings()
```

Comment:
This design requires the Additive object to keep a copy of all the server settings. Now that we've moved concurrent simulation management to the server, is there any reason to keep track of `nsims_per_server`? We can always get the latest server settings by calling the list settings endpoint.

Reply:
`nsims_per_server` no longer a member or property of `Additive`